src/raftstore/store/config.rs

```diff
@@ -197,6 +198,7 @@ impl Default for Config {
             use_delete_range: false,
             cleanup_import_sst_interval: ReadableDuration::minutes(10),
             local_read_batch_size: 1024,
+            local_read_concurrency: 2,
```

Comment:
I think set default value to 1 is better in most cases, it can reduce context switch.